src/update_handler.c

```diff
@@ -1733,6 +1802,7 @@ RaucUpdatePair updatepairs[] = {
 	{"*.ubifs", "ubivol", img_to_ubivol_handler},
 	{"*.ubifs", "ubifs", img_to_ubifs_handler},
 	{"*.img", "ext4", img_to_fs_handler},
+	{"*.img", "flash", img_to_flash_handler},
 	{"*.img", "nand", img_to_nand_handler},
```

Comment:
I'm not sure that `flash` is a good name, as NAND is also a type of flash. Maybe `nor`?

Reply:
Yes, this was my first thought, too. I'd also vote for 'nor'

Reply:
Sounds good to me.